lib/components-types-v1_0_x.ts

```diff
@@ -83,36 +83,62 @@ export interface ComponentsDefinitionV10X {
           /**
            * Dropdown with fixed number of options
            */
-          type?: "select";
-          options?: any[];
+          type: "select";
+          options: {
+            /**
+             * Label of the item
+             */
+            caption: string;
+            /**
+             * Value of the item
+             */
+            value?: string;
```

Comment:
Maybe good to add to the comment why you can leave the value out.

Reply:
Do keep in mind this file is generated from lib/components-schema-v1_0_x.ts, so the changes should go in there.